cmd/utils/flags.go

```diff
@@ -627,6 +627,11 @@ var (
 		Usage:    "odyssey test network: pre-configured proof-of-stake test network",
 		Category: flags.MiscCategory,
 	}
+	StoryFlag = &cli.BoolFlag{
+		Name:     "story",
```

Comment:
so it won't be called homer?

Reply:
No. The chain name is story and the name of the first release will be homer.